### ISSUE TITLE
[TC-531] Traffic Monitor does not pull new cr-config when all servers in in "ONLINE" and not "REPORTED" 

### ISSUE THREAD

**Issue by limited** [apache/incubator-trafficcontrol#989]
<p>When all cache servers are forced to be ONLINE, TM seems not to pull the cfg, with the below message in the log<br/>
"No REPORTED caches exist in Traffic Ops, nothing to poll."</p>

Author: Nir Sopher
JIRA Link: <a href="https://issues.apache.org/jira/browse/TC-531">https://issues.apache.org/jira/browse/TC-531</a>
Found Version: 2.1.0

**Comment by mitchell852** [comment]
I "think" this is expected behavior @nir-sopher . You shouldn't be setting caches to "online". Only caches marked as "reported" are actually monitored.

However, you might want to set "other" servers to online like riak servers or influx, etc. This is probably an artifact of an overloaded server table that contains both caches and non-caches....

maybe @rob05c has an opinion

**Comment by rob05c** [comment]
Yeah, caches should generally be set to REPORTED. Setting them ONLINE tells the Monitor to not poll, and just tell the Router that they're always available.

If you set all caches to ONLINE, the Monitor gives that error, because, well, it doesn't make sense to not monitor for health. If you really want to not monitor, and make that message go away, you could make a dummy cache with no Delivery Services assigned, and set that single dummy REPORTED and everything else ONLINE. But I don't recommend it.

**Comment by mitchell852** [comment]
@nir-sopher - is this still an issue?

**Comment by rawlinp** [comment]
Per recent comments in the ASF Slack channel #traffic-control this sounds like it's still an issue.
